frontend/src/metabase/querying/components/FilterPanel/utils.ts

```diff
@@ -1,11 +1,10 @@
 import * as Lib from "metabase-lib";
+import { getFilterStageIndexes } from "metabase-lib/v1/parameters/utils/targets";
```

Comment:
There should be no such dependency.,

Reply:
b14ea1bec7f392bd6c0dc2a4dc7a7ae630be741a